src/wordy/ast/BinaryExpressionNode.java

```diff
@@ -59,4 +66,46 @@ public String toString() {
     protected String describeAttributes() {
         return "(operator=" + operator + ')';
     }
+
```

Comment:
Your case writing is much more beautifully written than mine. A case statement like that with the arrows is a syntax that took me by surprise. I thought you needed to have break in the case statement which is why I thought about it and then avoided it